
syntax = "proto3";

import "sensor.proto";

package sensor_proto;

service CameraService {
    rpc SetParams(CameraParams) returns (Empty);
    rpc Init(CameraParams)      returns (Empty);
}

message CameraParams {
  SensorParams base = 1;
  CameraDims   dims = 2;
}

message CameraDims {
  int32 width   = 1;
  int32 height  = 2;
  int32 channel = 3;
}

